packages/api/tests/lib/test-ORCASearchCatalogQueue.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const test = require('ava');
+const nock = require('nock');
+const ORCASearchCatalogQueue = require('../../lib/ORCASearchCatalogQueue');
+
+test.before(() => {
+  nock.cleanAll();
```

Comment:
```suggestion
  nock.disableNetConnect();
  nock.cleanAll();
```

We should make sure that our unit tests don't actually connect to the external internet